src/main/java/frc/robot/commands/VisionDriverAid.java

```diff
@@ -25,19 +31,26 @@
   Supplier<Double> m_forwardVal;
   Supplier<Double> m_rotateVal;
   PIDController m_pid;
+  Command shooter;
+  Command feed;
+  Command shooterAuto;
 
   public VisionDriverAid(Joystick driveJoystick, Supplier<Double> rotateVal) {
     m_forwardVal = ()-> sign(Config.removeJoystickDeadband(driveJoystick.getRawAxis(Config.LEFT_CONTROL_STICK_Y)), Config.INVERT_FIRST_AXIS);
     // Use addRequirements() here to declare subsystem dependencies.
     addRequirements(DriveBaseHolder.getInstance());
     m_rotateVal = rotateVal;
     m_pid = new PIDController(0, 0, 0);
+    shooter = new SpinUpShooterWithTime(2000, 0);
+    shooterAuto = new AutomaticShooter(true);
+    feed = new WaitCommand(0.3).andThen(new ParallelRaceGroup(new IndexerForShooter(), new WaitCommand(1.5)).andThen(new InstantCommand(shooterAuto :: cancel)));
   }
    
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    shooter.schedule();
```

Comment:
Why are we scheduling the shooter here? I thought we'd use the autoShoot at the end of the command.